### PR TITLE
Switch tests to Test Mule

### PR DESCRIPTION
**Pull request by calebeby**
I switched the tests from Cypress to Test Mule. GitHub doesn't show the diff so it is probably helpful to open the old and new test files side by side.

### CODE REVIEW

**Review by tylersticka — COMMENTED** [review]
Deferring review to dev team when they have availability.

**Review by spaceninja — APPROVED** [review]
Code LGTM, and I don't see any obvious regressions on the preview environment.

Given the scope of this change, I'd appreciate you getting an approval from @gerardo-rodriguez or @Paul-Hebert as well before merging.

File: .style-dictionary/build.js
Comment (Paul-Hebert) [inline .style-dictionary/build.js:102]: Just FYI there is also the [`es6` format](https://amzn.github.io/style-dictionary/#/formats?id=javascriptes6)

This format returns each individual property as an export instead of exporting an object, so it may not work  here.

**Review by Paul-Hebert — APPROVED** [review]
Looks good! Thanks Caleb!

File: .style-dictionary/build.js
Comment (calebeby) [reply .style-dictionary/build.js:102]: Yeah I tried that and it wasn't working because lots of the patterns rely on nested objects/destructuring and the es6 format flattened them

**Pull request merged**
